doc/source/user_guide/io.rst

```diff
@@ -2072,6 +2072,10 @@ is ``None``. To explicitly force ``Series`` parsing, pass ``typ=series``
 * ``engine``: Either ``"ujson"``, the built-in JSON parser, or ``"pyarrow"`` which dispatches to pyarrow's ``pyarrow.json.read_json``.
   The ``"pyarrow"`` is only available when ``lines=True``
 
+.. warning::
+
+    Passing json literal strings will be deprecated in a future release of pandas.
```

Comment:
Can you move this to the `read_json` docstring instead?

Reply:
Should be fixed in latest push.